rating.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'file_handler'
+require_relative 'orders'
+
+class Rating
+  attr_reader :orders
+
+  def initialize(orders = FileHandler.new('orders').parse_file)
```

Comment:
it's better to pass just structure, retrieve a data from FileHandler isn't best idea, better to execute it before and just pass structure to Rating

Reply:
Fixed